eng/common/scripts/Create-APIReview.ps1

```diff
@@ -68,67 +75,82 @@ else
+        if ( ($SourceBranch -eq $DefaultBranch) -or (-not $version.IsPrerelease))
+        {
+            Write-Host "Submitting API Review for package $($pkg)"
+            $response = Submit-APIReview -packagename $pkg -filePath $pkgPath -uri $APIViewUri -apiKey $APIKey -apiLabel $APILabel
+            # HTTP status 200 means API is in approved status
+            if ($respCode -eq '200')
             {
+                Write-Host "API review is in approved status."
+            }
+            elseif ($version.IsPrerelease -or ($version.Major -eq 0))
+            {
+                # Ignore API review status for prerelease version
                 Write-Host "Package version is not GA. Ignoring API view approval status"
             }
-            elseif ($pkgInfo.SdkType -eq "client" -and $pkgInfo.IsNewSdk)
+            else
             {
-                $FoundFailure = $True
-                if ($respCode -eq '201')
+                # Return error code if status code is 201 for track2 data plane package
+                if ($pkgInfo.SdkType -eq "client" -and $pkgInfo.IsNewSdk)
                 {
-                    Write-Host "Package version $($version) is GA and automatic API Review is not yet approved for package $($PackageName)."
-                    Write-Host "Build and release is not allowed for GA package without API review approval."
-                    Write-Host "You will need to queue another build to proceed further after API review is approved"
-                    Write-Host "You can check http://aka.ms/azsdk/engsys/apireview/faq for more details on API Approval."
+                    if ($respCode -eq '201')
+                    {
+                        Write-Host "Package version $($version) is GA and automatic API Review is not yet approved for package $($PackageName)."
+                        Write-Host "Build and release is not allowed for GA package without API review approval."
+                        Write-Host "You will need to queue another build to proceed further after API review is approved"
+                        Write-Host "You can check http://aka.ms/azsdk/engsys/apireview/faq for more details on API Approval."
+                    }
+                    else
+                    {
+                        Write-Host "Failed to create API Review for package $($PackageName). Please reach out to Azure SDK engineering systems on teams channel and share this build details."
+                    }
+                    exit 1
                 }
                 else
                 {
-                    Write-Host "Failed to create API Review for package $($PackageName). Please reach out to Azure SDK engineering systems on teams channel and share this build details."
+                    Write-Host "API review is not approved for package $($PackageName). Management and track1 package can be released without API review approval."
```

Comment:
```suggestion
                    Write-Host "API review is not approved for package $($PackageName), however it is not required for this package type so it can still be released without API review approval."
```